Firestore/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Unreleased
+
+# v0.10.1
 - [fixed] Fixed a regression in Firebase iOS release 4.8.1 that could in certain
```

Comment:
nit: This should maybe refer to 4.8.2, since 4.8.1 was made unavailable following some critical analytics regressions.

Reply:
I'd vote to leave as is. Regression introduced in 4.8.1, continued in 4.8.2 and fixed in 4.9.0